tests/utils/test_dataset_combine.py

```diff
@@ -231,3 +232,53 @@ def test_combine_minari_version_specifiers(specifier_intersection, version_speci
     intersection = combine_minari_version_specifiers(version_specifiers)
 
     assert specifier_intersection == intersection
+
+
+# in the future, if the logic of save metadata of combined dataset changes, this should be changed as well
+def test_combine_dataset_with_different_metadata():
+    n_data = 2
+    dataset_list = []
+    for i in range(n_data):
+        dataset_id = f"cartpole-test-{i}-v0"
+        env = gym.make("CartPole-v1", max_episode_steps=500)
+        env = DataCollectorV0(env)
+        env.reset(seed=42)
+        for episode in range(5):
+            terminated = False
+            truncated = False
+            while not terminated and not truncated:
+                action = env.action_space.sample()  # User-defined policy function
+                _, _, terminated, truncated, _ = env.step(action)
+            env.reset()
+
+        # Create Minari dataset and store locally
+        dataset = minari.create_dataset_from_collector_env(
+            dataset_id=dataset_id,
+            collector_env=env,
+            algorithm_name="random_policy" + str(i),
+            code_permalink="https://github.com/Farama-Foundation/Minari/blob/f095bfe07f8dc6642082599e07779ec1dd9b2667/tutorials/LocalStorage/local_storage.py"
```

Comment:
can you change the permalink to be the following? https://github.com/Farama-Foundation/Minari/blob/main/tests/utils/test_dataset_combine.py